grisera_api/setup.py

```diff
@@ -0,0 +1,23 @@
+from channel.channel_service import ChannelService
+from channel.channel_model import ChannelIn
+import os
+from time import sleep
+
+
+class SetupNodes:
+    """
+    Class to init nodes in grahp database
+    """
+    def set_channels(self):
+        """
+        Initialize values of channels
+        """
+        channel_service = ChannelService()
+        channel_types = ["Audio", "BVP", "Chest size", "Depth video", "ECG",
+                         "EDA", "EEG", "EMG", "RGB video", "Temperature"]
+
+        if not os.path.exists("pipe"):
+            open("pipe", "w").write("Busy")
+            sleep(30)
+            [channel_service.save_channel(ChannelIn(type=channel_type)) for channel_type in channel_types]
+            os.remove("pipe")
```

Comment:
I don't follow this... 
1. I believe it is a locking mechanism? If so, why the file is called `pipe`?
2. If the `pipe` file exists, we do nothing?
3. Why we are sleeping 30s?
4. Don't we have to check if the channels are already created in the DB?
5. Do we assume that the list of channels types is immutable?

Reply:
1. I referred to pipes mechanism, but it could be change to something like "lock".
2. Yes, because we want to inform other workers that someone is doing it.
3. It's to give database time to start.
4. I was thinking about that, with this step we should do new endpoint in graph api. I can make it for purpose of this task.
5. According to product backlog it's not said if we will make new channel types. Scope of the task was to create channel with given types.

Reply:
Thanks for explanation :)
1. Lets name this file `lock`
4. Yes, I think it's necessary to check if these channels are already created. We can restart application, without clearing DB. So with each restart of application, channels will be duplicated (if I understand it correctly).